src/cli/analytics.rs

```diff
@@ -164,11 +167,60 @@ fn run(
             inner: vec![],
         })?;
 
+        if with_meta {
+            let mut query_results = vec![];
+            while let Some(mut query_result) = rt
+                .block_on(async { json_streamer.read_row().await })
+                .map_err(|e| ShellError::GenericError {
+                    error: format!("failed to read analytics query result: {}", e),
+                    msg: "".to_string(),
+                    span: None,
+                    help: None,
+                    inner: vec![],
+                })?
+            {
+                if !query_results.is_empty() {
+                    query_results.append(&mut vec![b','])
+                }
+                query_results.append(&mut query_result)
+            }
+
+            let epilog = rt
+                .block_on(async { json_streamer.read_epilog().await })
+                .map_err(|e| ShellError::GenericError {
+                    error: format!("failed to read stream epilog: {}", e),
+                    msg: "".to_string(),
+                    span: None,
+                    help: None,
+                    inner: vec![],
+                })?;
+
+            let (start, _) = from_utf8(&epilog).unwrap().split_at(epilog.len() - 1);
```

Comment:
Is there no way that we can create the nu value without manually creating a JSON value?